### PR TITLE
Use framework.ExpectNoError() for e2e/node

### PR DESCRIPTION
**Pull request by s-ito-ts**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Replaces Expect(err).NotTo(HaveOccurred()) with ExpectNoError() in e2e/node.

**Which issue(s) this PR fixes**:
Ref: #77103

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```
/sig testing

### CODE REVIEW

**Comment by draveness** [comment]
cc/ @oomichi

File: test/e2e/node/kubelet_perf.go [hunk 1]
```diff
@@ -31,7 +31,6 @@ import (
 	imageutils "k8s.io/kubernetes/test/utils/image"
 
 	"github.com/onsi/ginkgo"
-	"github.com/onsi/gomega"
```
Comment (oomichi) [inline test/e2e/node/kubelet_perf.go]: ++

**Review by oomichi — COMMENTED** [review]
/lgtm

**Pull request merged**